src/data/02-programming-languages/fibjs.yml

```diff
@@ -0,0 +1,11 @@
+name: 'FibJS'
+homepageURL: 'https://fibjs.org'
+repoURL: 'https://github.com/fibjs/fibjs'
+portingEfforts:
+  - authors: ['pinelliaw']
```

Comment:
You should add the actual contributor(s) to both `porters.yml` (find in the `src/data` directory yourself) and here, or IIRC the build will fail.